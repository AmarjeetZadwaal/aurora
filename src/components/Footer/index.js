import React from "react";
import PropTypes from "prop-types";
import styled from "styled-components";

import colors from "../../theme/colors";
import Container from "../Grid/Container";

const Wrapper = styled.div`
  width: 100%;
  background-color: ${colors.blackPearl};
`;

const Footer = ({ children }) => (
  <Wrapper>
    <Container>{children}</Container>
  </Wrapper>
);

Footer.propTypes = {
  children: PropTypes.node.isRequired
};

export default Footer;
