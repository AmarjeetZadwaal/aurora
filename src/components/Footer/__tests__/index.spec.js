import React from "react";
import { render } from "react-testing-library";

import Footer from "../index";

describe("<Footer />", () => {
  it("renders Footer correctly", () => {
    const { container } = render(
      <Footer>
        <div>Los Angeles</div>
        <div>New York</div>
        <div>Maimi</div>
      </Footer>
    );
    expect(container.firstChild).toMatchSnapshot();
  });
});
