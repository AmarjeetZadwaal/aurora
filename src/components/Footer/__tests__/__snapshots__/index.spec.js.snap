// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer /> renders Footer correctly 1`] = `
<div
  class="sc-bwzfXH kFjJta"
>
  <div
    class="sc-bdVaJa eFJsIk"
  >
    <div>
      Los Angeles
    </div>
    <div>
      New York
    </div>
    <div>
      Maimi
    </div>
  </div>
</div>
`;
